openlcs/libs/driver.py

```diff
@@ -140,7 +140,7 @@ def get_paginated_data(self, url, query_params=None):
                 # FIXME: fail silently is not ideal
                 break
 
-    def post(self, url, data, timeout=10):
+    def post(self, url, data, timeout=300):
```

Comment:
With setting the default, the timeout on `save_componnets` and `filter_duplicate_import` could also be removed, there would be tons of data with following products loading.

Reply:
OK, all of them  use the default setting `300s`